jme3-examples/src/main/java/jme3test/terrain/PBRTerrainAdvancedTest.java

```diff
@@ -0,0 +1,395 @@
+package jme3test.terrain;
```

Comment:
needs a JME copyright notice and license

Reply:
I added the copyright notice from another test case, let me know if it looks alright